src/routingComps/MyProjects.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+import { Button, Card } from 'react-bootstrap';
+
+export default class MyProjects extends Component {
+	constructor(props) {
+	    super(props);
+	    this.state = {
+	    	projects:[],
+	    	username: null
+	    };
+	 }
+	  
+	componentDidMount() {
+		// Getting current users' username.
+		axios.get('http://localhost:5000/users/getinfo', {withCredentials: true}) 
+			.then(response => {
+				this.setState({
+					username: response.data.username
+				})
+			})
+			.catch(function (error) {
+		        console.log(error);
+		    })
+		// Fetching all users' projects.
+		axios.get('http://localhost:5000/projects/getinfo', {withCredentials: true}) 
+		    .then(response => {
+		        this.setState({projects: response.data})
+	        })
+		    .catch(function (error) {
+		        console.log(error);
+		    })
+	}
+	// Creating a card component for each displayed project.
```

Comment:
Is that all? what else will be displayed on the page? where can the users submit their projects and see more details about them?

Reply:
yeah the aim of this page is to list all the projects joined by the current user. I am working on Projects page where users will be able to see ones submissions and be able to submit their work there.

Reply:
Please mention that in the comments so that it is understood

Reply:
Done.